OrbitCore/Pdb.h

```diff
@@ -34,7 +34,8 @@ class Pdb {
 
   void PopulateFunctionMap();
   void PopulateStringFunctionMap();
-  void ApplyPreset(const orbit_client_protos::PresetFile& preset);
+  [[nodiscard]] std::vector<orbit_client_protos::FunctionInfo*>
+  FunctionsToSelect(const orbit_client_protos::PresetFile& preset) const;
```

Comment:
GetSelectedFunctions or GetSelectedFunctionsFromPreset

Reply:
Ah was thinking about the name a lot. thanks for that suggestion. I went with the latter.